Refresher/UI/PSPSetupForm.cs

```diff
@@ -26,18 +26,38 @@ public PSPSetupForm() : base("PSP Setup")
         
         foreach (DriveInfo drive in drives)
         {
+            bool isVita = false;
+            string pspEmuFolderName = "";
+
             try
             {
                 Console.WriteLine($"Checking drive {drive.Name}...");
-                
-                //If theres no PSP folder,
-                if (!Directory.Exists(Path.Combine(drive.RootDirectory.FullName, "PSP")))
+
+                // Match for all directories called PSP and PSPEMU
+                // NOTE: we do this because the PSP filesystem is case insensitive, and the .NET STL is case sensitive on linux
```

Comment:
Does this really matter - e.g. `PSP` could be lowercase and `pspemu` can be uppercase?

Reply:
Apparently actually this *can* happen on the PSP and cause issues particularly on Linux - this code makes sense now.